### PR TITLE
GitOps Service: Set GitOps Service replicas to 0 (GITOPSRVCE-801)

### PR DESCRIPTION
**Pull request by jgwest**
JIRA:
https://issues.redhat.com/browse/GITOPSRVCE-801

### CODE REVIEW

File: components/gitops/production/base/kustomization.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- https://github.com/redhat-appstudio/managed-gitops/manifests/overlays/stonesoup-member-cluster?ref=87e1f9acc67bf033d2159951ba6489f0836586ef
+- https://github.com/redhat-appstudio/managed-gitops/manifests/overlays/stonesoup-member-cluster?ref=006efe22e41e6c249d333e0a0f2a87bd32d3bf02
```
Comment (gbenhaim) [inline components/gitops/production/base/kustomization.yaml:5]: can we change dev and stage first and only then prod?

**Comment by jgwest** [comment]
Removed production, and rebased to laster from `main` branch.

**Comment by gbenhaim** [comment]
replaced by https://github.com/redhat-appstudio/infra-deployments/pull/3760

**Pull request closed without merging**